packages/google-cloud-serverless/test/gcpfunction/events.test.ts

```diff
@@ -1,4 +1,4 @@
-import { SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, SEMANTIC_ATTRIBUTE_SENTRY_SOURCE } from '@sentry/core';
+import { SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, SEMANTIC_ATTRIBUTE_SENTRY_SOURCE, isThenable } from '@sentry/core';
```

Comment:
I believe isThenable is not used.

also thank you for this simplification - I've missed the fact that _wrapEventFunction calls callback and no extra Promise waiting needed.

Reply:
should I just add a commit to fix biome warning? not sure about squashing at this point though.

Reply:
ah ok - thanks for clarification. pushed a fix